src/db/timesheets/ItemsOperations.ts

```diff
@@ -0,0 +1,221 @@
+import {TimeSheetSchema, TimesheetEntrySchema, ScheduleEntrySchema, NoteSchema} from '../schemas/Timesheet'
+import {UpdateRequest, InsertRequest, DeleteRequest, TimesheetListItems} from '../schemas/UpdateTimesheet'
+import { ExceptionsHandler } from '@nestjs/core/exceptions/exceptions-handler';
+//Not sure why but only works if imported like this :| 
+const moment = require('moment-timezone'); 
+
+/* Interface holding all operations available for a field in the timesheet that should support 
+    updates, inserts, and deletions. 
+*/
+interface ItemsOperations {
```

Comment:
I really like this! In the future, maybe there could be some more shared code (looks like the Delete and Update functions tend to be fairly similar), but for now I think it looks really good!